### PR TITLE
feat: ticket creation updated

### PR DESCRIPTION
**Pull request by boxmein**
Updates the ticket creation API to use the new version of the ticket creation endpoint.

Usage:

```typescript
const { id } = await Qminder.Ticket.create({
  lineId: '202921',
  firstName: 'James',
  lastName: 'Baxter',
});
console.log(id); // example: "1902949"
```

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

**Comment by boxmein** [comment]
Initial feedback from @KarlMae :

> The interface should be agreed upon by other devs as well. Maybe `Qminder.TicketV2.create` or `Qminder.Ticket.V2.create` or `Qminder.V2.Ticket.create` or a parameter like `Qminder.Ticket.create(version = 2)` should be proposed as alternatives.

> Consider splitting up into files if there are multiple interfaces in a single file

**Review by 2ndalpha — APPROVED** [review]

File: src/lib/model/ticket/input-field-request.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Set an input field of a ticket.
+ *
+ * One of 2 options are allowed:
+ * (inputFieldId, value) or (inputFieldId, optionIds)
+ */
+export type InputFieldRequest =
+  | ValueInputFieldRequest
+  | OptionIdsInputFieldRequest;
+
+export interface BaseInputFieldRequest {
```
Comment (KarlMae) [inline src/lib/model/ticket/input-field-request.ts:13]: BaseInputFieldRequest probably should not be exported? It's not exposed in the public api and not used anywhere else

**Review by KarlMae — CHANGES_REQUESTED** [review]
One small question to be answered. Otherwise looks nice 👍

File: src/lib/model/ticket/input-field-request.ts (hunk 1 above)
Comment (boxmein) [reply src/lib/model/ticket/input-field-request.ts]: Fixed it!

**Review by KarlMae — APPROVED** [review]
💯

**Pull request merged**